doc/cabal-project.rst

```diff
@@ -218,8 +220,11 @@ version control.
         tag: e76fdc753e660dfa615af6c8b6a2ad9ddf6afe70
         post-checkout-command: autoreconf -i
 
-cabal-install 3.4 sdists the ``source-repository-package`` repositories and uses resulting tarballs as project packages.
-This allows sharing of packages across different projects.
+Since version 3.4, cabal-install creates tarballs for each package coming
+from a source-repository-package stanza (effectively applying cabal
```

Comment:
The name of the stanza should be in double ticks. Same with cabal sdist. Same applies above.